depop-app/src/components/ProductsContainer.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import {ProductCard} from "./ProductCard";
+import {useGetProducts} from "../hooks/useGetProducts";
+import "./ProductsContainer.css"
+
+export const ProductsContainer = () => {
+
+    const {products, loading} = useGetProducts('https://5c78274f6810ec00148d0ff1.mockapi.io/api/v1/products');
+
+    return (
+        <React.Fragment>
+            {!loading && products &&
+                <div className={"subTitle"}><b>{products.length} Results</b></div>
+            }
+            {loading && <span>Loading items</span>}
```

Comment:
D'ya reckon we should remove this to see if they implement it?

Reply:
Good question. I guess it's a pretty one to quickly implement to get someone building confidence. I could leave the variable in the hook, but just not import it yet.